### ISSUE TITLE
Escrever em português ou inglês

### ISSUE THREAD

**Issue by talyssonoc** [LFeh/poste-mais#42]
Há quase um mês tenho pensado em começar um blog, falando sobre coisas que venho aprendido (principalmente no que tange ao NodeJS, mas não limitado a isso), pois também sinto falta de mais devs brasileiros escrevendo sobre desenvolvimento, boas práticas de programação e tudo mais, porém uma coisa que me faz pensar e ainda não me permitiu começar é a dúvida: escrever em português ou em inglês ?

Começando pelo fato de que faltam blogs brasileiros, português seria uma boa idéia, mas parece, por algum motivo que não sei ainda, que pra mim faz mais sentido escrever em inglês, o que vocês acham ?

**Comment by fdaciuk** [comment]
Penso que a resposta para essa pergunta deve ser: qual público você quer alcançar?
Ao começar um blog, você quer que alguém o leia. Quem são essas pessoas? Brasileiros? O mundo todo? Iniciantes? Manjadores dos paranauê?

Respondendo isso, acho que fica mais fácil tomar a decisão :)

**Comment by talyssonoc** [comment]
Como eu quero falar de assuntos que eu queria achar blogs no mesmo estilo quando eu estava começando, acho que o público seria brasileiros (mas nesse caso estaria escrevendo em português pra ser algo brasileiro, mas não muda minha opinião de que pessoas da computação deveriam saber inglês, que é uma das coisas que me faz querer escrever em inglês).

O público seria de iniciantes a pessoas com mais interesse, que estão começando a avançar e aprender coisas não básicas... mas não sei se isso chega a ser um "motivo" pra escrever em português, daí minha dúvida !

**Comment by fdaciuk** [comment]
Entendi.. mas assim.. se for pra escrever aleatóriamente, só pra desabafar mesmo, acho que é válido em qualquer língua. Talvez pra melhorar o inglês, ou em pt-BR mesmo, não vai fazer diferença. Faça como vocẽ se sentir melhor.

Mas a realidade é que muitos brasileiros ainda se batem com o inglês, ainda que isso seja o básico hoje em dia. Por isso acho que você precisa definir bem o público do seu blog, conforme a realidade que temos hoje. Se você quer que brasileiros consumam seu conteúdo, mas escreve em inglês, saiba que a minoria é que vai aproveitar, infelizmente. =(

**Comment by talyssonoc** [comment]
Não, é sobre dev que pretendo escrever mesmo, não um blog pessoal (acabei reparando que na minha resposta eu não citei nenhuma vez esse fato, foi mal !).

E sim, concordo que ainda tem muita gente que cria uma barreira nisso, fica boa parte da vida falando "precisa aprender inglês", "tenho que me aprofundar", mas fica sempre com receio (geralmente com a desculpa de ser por "preguiça de traduzir") de pegar algo em inglês pra ler ! Decidi que será mesmo em português ! Em breve estarei pegando alguma issue pra escrever sobre ! =)

Mas ainda concordo com a continuidade dessa issue de discussão sobre o tema !

**Comment by LFeh** [comment]
Então @talyssonoc. Obviamente o inglês é super importante na nossa profissão. E pelo menos a leitura técnica deveria ser obrigatório para qualquer desenvolvedor.

Como o D2K disse, depende muito do que você quer. Acredito que qualquer problema deve ser feito em inglês. Mas quando se trata de postagens, acredito que é um pouco mais amplo.

Pode por exemplo, escrever em ptBR no seu blog, e usar o Medium para escrever textos mais 'pessoais' em inglês, eu faço assim (apesar de ter escrito apenas 1 vez no Medium :( )

**Comment by jugoncalves** [comment]
Eu escrevo em inglês para maximizar meu tempo. Se preferirem não ler por causa disso, não tem muito que eu possa fazer, na verdade. :disappointed:

**Comment by LFeh** [comment]
@jugoncalves Concordo totalmente. Alias, quem nem tentar ler por causa disso, provavelmente também não leria se fosse em ptBR :)

**Comment by felipesousa** [comment]
@talyssonoc Brother, falo como leitor, creio que escrever em ingles vai mostrar para iniciantes que é mais do que necessário começar a aprender inglês, visto que o idioma é super necessário para se dar bem tanto na profissão como e qlqr outra.. Eu vejo que é importante tanto para atingir um público maior quanto para incentivar os mais novos no ramo a se dedicarem ao idioma.

Espero ter ajudado, boa sorte com o blog :)

**Comment by douglascamata** [comment]
Bom, minha opinião como leitor não conta muito porque eu falo inglês. Mas tenho amigos de todos os níveis que não falam/leem muito bem inglês. Então decidi que no novo blog que estou construindo com `middleman` eu vou escrever TODOS os posts em inglês e português e usar algumas extensão pra facilitar o i18n por motivos de:

1. Escrevendo nos dois idiomas eu estarei ensinando a mim mesmo, então de maneira alguma será uma perda de tempo (acho até necessário para mim, porque já não trabalho no Brasil e não pretendo voltar);
2. Estarei ajudando mais iniciantes e/ou pessoas com dificuldades;
3. Estarei aumentando meu networking consideravelmente, o que é sempre bom.

**Comment by sergiokopplin** [comment]
Escrever em inglês é bom para fins de estudo, para alcanças mais pessoas e também para se adaptar ao mundo da tecnologia.
Depende realmente do seu público, mas eu gosto muito de ver brasileiros escrevendo em inglês, eu dou uma valorizada! ;)
